wpilibj/src/main/java/edu/wpi/first/wpilibj/SensorUtil.java

```diff
@@ -17,64 +17,61 @@
 import edu.wpi.first.wpilibj.hal.SolenoidJNI;
 
 /**
- * Base class for all sensors. Stores most recent status information as well as containing utility
- * functions for checking channels and error processing.
+ * Stores most recent status information as well as containing utility functions for checking
+ * channels and error processing.
  */
-public abstract class SensorBase extends SendableBase {
+public class SensorUtil {
```

Comment:
Should this be final with a private constructor?

Reply:
And likewise for C++?